tools/pipeline_perf_test/test_suites/integration/templates/configs/engine/backpressure/otap-attr-otap.yaml

```diff
@@ -0,0 +1,43 @@
+settings:
+  default_pipeline_ctrl_msg_channel_size: 100
+  default_node_ctrl_msg_channel_size: 100
+  default_pdata_channel_size: 100
+
+nodes:
+  receiver:
+    kind: receiver
+    plugin_urn: "urn:otel:otap:receiver"
+    out_ports:
+      out_port:
+        destinations:
+          - attr
+        dispatch_strategy: round_robin
+    config:
+      listening_addr: "0.0.0.0:4317"
+      message_size: 100
+      compression_method: zstd
+      wait_for_result: true
```

Comment:
I was looking for a wait_for_result=false test. 

We made wait_for_result=false the default: should this test the other way?

#1328